src/cli-newtonraphson.cpp

```diff
@@ -0,0 +1,52 @@
+// this C++ snippet is stored as src/newtonraphson.cpp
```

Comment:
this is exactly the same as src/cgi-newtonraphson because generated by <<algorithm>>. I suggest to make all these changes in  <<algorithm>>.